LongWarOfTheChosen/Config/XComGameData_WeaponData.ini

```diff
@@ -1768,3 +1768,210 @@ SPARK_NOVA_SELF_BASEDAMAGE= (Damage=0, Spread = 0, PlusOne = 0, Crit = 0, Pierce
 FERALMEC_M1_PROXY_WPN_BASEDAMAGE=(Damage=2, Spread=0, PlusOne=25, Crit=1, Pierce=0, Shred=0, Tag="", DamageType="Projectile_Conventional")
 
 SECTOPOD_MARKOV_PROXY_WPN_BASEDAMAGE=(Damage=3, Spread = 0, PlusOne = 25, Crit = 1, Pierce = 0, Shred = 1, Tag = "", DamageType="Projectile_Conventional")
+
+[WotCWeaponsAndArmor.X2Item_Weapons]
+;Bukllpup stats
+BULLPUP_CONVENTIONAL_BASEDAMAGE=(Damage=3, Spread=0, PlusOne=50, Crit=1, Pierce=0, Shred=0, Tag="", DamageType="Projectile_Conventional")
+BULLPUP_MAGNETIC_BASEDAMAGE=(Damage=6, Spread=0, PlusOne=50, Crit=2, Pierce=0, Shred=0, Tag="", DamageType="Projectile_MagXCom")
```

Comment:
This is one more damage than the SMG, whereas the other tiers are the same as SMG. Intentional? Also, is it intentional to have no spread on this weapons? I guess that's what the PlusOne is for. It makes the range effectively 5-6.

Hmmm...not sure what to do about the faction weapons in the long term. Feel like they need to be more distinctive.

Reply:
So as far as the faction weapons go, they do not have a spread. That is intended and the only thing I tweaked there is the base damage. I made them higher so that they can deal with the scaling health enemies as the elite enemies have quite a bit more hp than in vanilla.

I am going to have have bullpups sit at the lowest damage, vektor rifles sit around the middle and have templars have the highest as they require more risk.

Not sure about what the scondaries. I might just keep them as they are tbh. Will have to have a secondary discussion about that.